TeXmacs/doc/main/links/man-multifile.zh.tm

```diff
@@ -0,0 +1,36 @@
+
+  \<#5728\>\<#5199\>\<#4E00\>\<#672C\>\<#4E66\>\<#65F6\>\<#FF0C\>\<#4EBA\>\<#4EEC\>\<#901A\>\<#5E38\>\<#5C06\>\<#4F1A\>\<#5404\>\<#4E2A\>\<#7AE0\>\<#8282\>\<#653E\>\<#5728\>\<#6587\>\<#4EF6\>
+  <verbatim|c1.tm>\<#3001\><verbatim|c2.tm> \<#76F4\>\<#5230\>
+  <verbatim|c1.tm> \<#4E2D\>\<#3002\>\<#4E0B\>\<#4E00\>\<#6B65\>\<#662F\>\<#4E3A\>\<#6574\>\<#672C\>\<#4E66\>\<#521B\>\<#5EFA\>\<#4E00\>\<#4E2A\>\<#6587\>\<#4EF6\>
+  <verbatim|book.tm> \<#FF0C\>\<#5E76\>\<#5728\>\<#5176\>\<#4E2D\>\<#4F7F\>\<#7528\>\<#4E0A\>\<#8FF0\>\<#673A\>\<#5236\>\<#6765\>\<#5305\>\<#542B\>
+  <verbatim|c1.tm>\<#3001\><verbatim|c2.tm> \<#76F4\>\<#5230\>
+  <verbatim|c1.tm>\<#3002\>\<#76EE\>\<#5F55\>\<#3001\>\<#53C2\>\<#8003\>\<#6587\>\<#732E\>\<#7B49\>\<#901A\>\<#5E38\>\<#4F1A\>\<#653E\>\<#5728\>
+  <verbatim|book.tm> \<#4E2D\>\<#3002\>
+
+  \<#4E3A\>\<#4E86\>\<#5728\>\<#7F16\>\<#8F91\>\<#7279\>\<#5B9A\>\<#7AE0\>\<#8282\>
+  ci.tm \<#65F6\>\<#67E5\>\<#770B\>\<#4E0E\>\<#5176\>\<#4ED6\>\<#7AE0\>\<#8282\>\<#7684\>\<#4EA4\>\<#53C9\>\<#5F15\>\<#7528\>\<#FF0C\>\<#53EF\>\<#4EE5\>\<#4F7F\>\<#7528\><menu|Tools|Project|Attach
+  master>\<#5C06\> <verbatim|book.tm> \ \<#6307\>\<#5B9A\>\<#4E3A\>\<#6587\>\<#4EF6\>
+  <verbatim|c1.tm> \<#5230\> <verbatim|cn.tm>
+  \<#7684\>\P\<#4E3B\>\<#6587\>\<#4EF6\>\Q\<#3002\>\<#5728\>\<#76EE\>\<#524D\>\<#FF0C\>\<#8BE5\>\<#673A\>\<#5236\>\<#4E0D\>\<#4F1A\>\<#5904\>\<#7406\>\<#7AE0\>\<#8282\>\<#7F16\>\<#53F7\>\<#672C\>\<#8EAB\>\<#FF0C\>\<#56E0\>\<#6B64\>\<#60A8\>\<#53EF\>\<#80FD\>\<#9700\>\<#8981\>\<#5728\>\<#6BCF\>\<#4E2A\>\<#7AE0\>\<#8282\>\<#6587\>\<#4EF6\>\<#7684\>\<#5F00\>\<#5934\>\<#624B\>\<#52A8\>\<#5206\>\<#914D\>\<#73AF\>\<#5883\>\<#53D8\>\<#91CF\>
+  <src-var|chapter-nr> \<#4EE5\>\<#4FBF\>\<#5728\>\<#7F16\>\<#8F91\>\<#65F6\>\<#83B7\>\<#5F97\>\<#6B63\>\<#786E\>\<#7684\>\<#7F16\>\<#53F7\>\<#3002\>
+
+  <tmdoc-copyright|1998\U2002|Joris van der Hoeven>
```

Comment:
update the copyright to 1998-2022, and add your name

for translation, the author should be the original author and also the translator.